frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectFeatureTogglesHeader/CreateFeatureDialog.tsx

```diff
@@ -103,6 +106,15 @@ const CreateFeatureDialogContent = ({
     const navigate = useNavigate();
     const openFeatureCreatedFeedback = useFeatureCreatedFeedback();
 
+    const [storedFlagConfig, storeFlagConfig] =
```

Comment:
this hook is modelled after useState that normally returns a tuple [val, setVal] with an explicit value and a setter. The naming convention here confused me since it's not very idiomatic and I thought it's the value part listed twice.